---
geometry_msgs/Quaternion angle_lock